data/default/health/service-registry.json

```diff
@@ -6,110 +6,132 @@
       "service": "USER",
       "actions": [
         {
-          "login": "/user/oauth/token"
+          "name": "login",
```

Comment:
Since this is an `actions` array, let's rename `name` to `action`. We can also add in the `entityName` we were talking about yesterday which is required to populate the request object.

Reply:
That pattern fits for the services we have in health but might not work for other existing/future DIGIT services so you've hit the nail on the head in terms of the aim of future proofing the design as the overall vision for the flutter app is to evolve into a generic DIGIT app.